api/src/cli/utils/create-env/env-stub.liquid

```diff
@@ -113,6 +113,15 @@ PUBLIC_URL="/"
 # DB_USER="root"
 # DB_PASSWORD="secret"
 
+####################################################################################################
+### Global Redis
+
+# These settings will be used as fallback for RATE_LIMITER_REDIS_*,
+#  CACHE_REDIS_*, MESSENGER_REDIS_*
```

Comment:
```suggestion
# CACHE_REDIS_*, MESSENGER_REDIS_*
```

(or maybe we can turn this into a single line comment since technically there are other comments that are longer)